docs/deployment/recovery/index.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: "Recovering from failure"
+---
+
+If you run into an error when operating wasmCloud, the first step is to diagnose the problem. Typically, this will mean checking three sources of information:
+
+* Logs for the [wasmCloud host](/docs/concepts/hosts.mdx) process
+* Logs for the [wasmCloud Application Deployment Manager (`wadm`)](/docs/ecosystem/wadm/index.md) process
+* Activity on [NATS](/docs/ecosystem/nats/index.mdx)
+
+## Troubleshooting on Kubernetes
+
+When deploying [wasmCloud on Kubernetes](/docs/deployment/k8s/index.mdx), you can access logs for wasmCloud hosts and `wadm` with `kubectl`. 
+
+To check logs for `wadm`:
+
+```shell
+kubectl logs -f -l app.kubernetes.io/instance=wadm -c wadm
```

Comment:
This serves as a good reminder that we should probably set up a static label that we can count on being there regardless of what a given deployment or application is called when deployed via helm charts.

I think for the moment this is fine, it just might be a little bit surprising for folks if they name their deployment of wadm differently.

Reply:
I should note that the `-c wadm` isn't needed here as `wadm` is the only container defined in the deployment/pod, which means `kubectl` will default to it.